### DISCUSSION TITLE
Alpine Expression Error: func(...).catch is not a function after upgrade from 3.4 to 3.5

### DISCUSSION THREAD

**Discussion by christianwiedemann** [alpinejs/alpine#2359] [7. Bugs (archived)] [answered]
Hi,

I am using alpinejs inside a node module like:

``` 
 import Alpine from 'alpinejs';

window.Alpine = Alpine;
Drupal.behaviors.alpinejs = {
  isStarted: false,
  attach() {
    if (Drupal.behaviors.alpinejs.isStarted === false) {
      Alpine.start();
      Drupal.behaviors.alpinejs.isStarted = true;
    }
  },
};
``` 

This works fine until 3.5. After upgrading to 3.5 I get `Alpine Expression Error: func(...).catch is not a function` 

I tried to debug that because I think this is propably a problem of my node configuration but can't find any reason. 

Maybe someone can help out here.

**Comment by danddanddand** [comment, marked as answer]
You need to ensure your transpile target is es2016 or higher to have proper async support

**Reply by ftes** [reply]
I even had to change it to es2017 (using esbuild).

**Comment by christianwiedemann** [comment]
Thanks!
